examples/platform/linux/BUILD.gn

```diff
@@ -36,6 +38,10 @@ source_set("app-main") {
   if (chip_enable_pw_rpc) {
     defines += [ "PW_RPC_ENABLED" ]
   }
+  if (chip_ip_commissioning) {
+    # BLE and on-network. Code defaults to BLE if this is not set.
+    defines += [ "CONFIG_RENDEZVOUS_MODE=6" ]
```

Comment:
It is not mandatory for commissioners to use this bitmask. In fact, since this bitmask is absent from numerical pairing code, it is merely a hint.

Overall, the commissioner examples should just "look everywhere they can" for devices to be commissioned. The onboarding payload does enable filtering down with additional knowledge to reduce possibility of discriminator collision. 

Is there an issue tracking this?

Reply:
From discussion offline - the codes are not optional in the QR code. This flag doesn't control anything on-device. Right now, the commissioner example code (chip-device-ctrl) has a direct IP and a direct BLE connection, but also supports connecting from the QR code, which pulls the setup pin, discriminator and rendezvous flags. The QR code pairing example will only check the flags in the QR code, as these are mandatory. An implementation of a manual pairing in the example controller would include all, but has not been written yet.